### PR TITLE
Fix/cta size

### PR DESCRIPTION
**Pull request by luiskcs89**
Task: https://openforge.teamwork.com/#/tasks/16321502

### CODE REVIEW

**Comment by Wholloran** [comment]
QA passes

File: src/components/app-cta/app-cta.scss [hunk 1]
```diff
@@ -1,13 +1,17 @@
 .cta {
   height: 344px;
+  background: #008092;
```
Comment (PaulPaulDevelops) [inline src/components/app-cta/app-cta.scss]: Is this a new color in the style guide? We should add it globally if we are going to start using it.

Comment (luiskcs89) [reply src/components/app-cta/app-cta.scss]: yes, moved to a variable now in this PR, I guess it should be done in all the others that use this color now too.

**Comment by luiskcs89** [comment]
Addressed @PaulPaulDevelops comment and fixed conflicts

**Pull request merged**